pandas/tests/series/test_constructors.py

```diff
@@ -1168,10 +1180,9 @@ def test_constructor_with_datetime_tz3(self):
 
     def test_constructor_with_datetime_tz2(self):
         # with all NaT
-        ser = Series(NaT, index=[0, 1], dtype="datetime64[ns, US/Eastern]")
-        dti = DatetimeIndex(["NaT", "NaT"], tz="US/Eastern").as_unit("ns")
-        expected = Series(dti)
-        tm.assert_series_equal(ser, expected)
+        s = Series(NaT, index=[0, 1], dtype="datetime64[ns, US/Eastern]")
```

Comment:
looks like some stuff got lost in a rebase, s->ser, as_unit (can just use dtype instead of tz+as_unit)

Reply:
fixed